packages/react-utilities/src/compose/resolveShorthand.ts

```diff
@@ -1,36 +1,40 @@
 import { isValidElement } from 'react';
-import type { UnknownSlotProps, ShorthandProps } from './types';
+import type { UnknownSlotProps, SlotShorthandValue, ReplaceNullWithUndefined } from './types';
```

Comment:
nit: Let's keep these ordered

```suggestion
import type { ReplaceNullWithUndefined, SlotShorthandValue, UnknownSlotProps } from './types';
```